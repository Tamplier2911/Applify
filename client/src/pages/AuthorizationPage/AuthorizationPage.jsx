import "./AuthorizationPage.scss";
import React from "react";

const AuthorizationPage = () => {
  return (
    <div>
      <div>Authorization Page</div>
    </div>
  );
};

export default AuthorizationPage;
