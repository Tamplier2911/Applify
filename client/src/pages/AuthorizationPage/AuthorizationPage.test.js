import React from "react";
import toJson from "enzyme-to-json";

// component
import AuthorizationPage from "./AuthorizationPage";

// utils
import { setShallow, findByTestAttr } from "../../tests/testUtils";

describe("<AuthorizationPage />", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = setShallow(<AuthorizationPage />);
  });

  it("renders without error", () => {
    const authPage = findByTestAttr(wrapper, "auth-page");
    expect(authPage.length).toBe(1);
  });

  it("renders childrens", () => {
    const signIn = findByTestAttr(wrapper, "auth-page-signin");
    expect(signIn.length).toBe(1);

    const signUp = findByTestAttr(wrapper, "auth-page-signup");
    expect(signUp.length).toBe(1);
  });

  it("matching snapshot", () => {
    const authPage = findByTestAttr(wrapper, "auth-page");
    expect(toJson(authPage)).toMatchSnapshot();
  });
});
