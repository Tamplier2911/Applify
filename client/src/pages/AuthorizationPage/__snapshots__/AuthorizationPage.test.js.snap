// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthorizationPage /> matching snapshot 1`] = `
<styled.section
  data-test="auth-page"
>
  <styled.div />
  <Connect(SignIn)
    data-test="auth-page-signin"
  />
  <Connect(SignUp)
    data-test="auth-page-signup"
  />
</styled.section>
`;
